renderer/components/Editor.tsx

```diff
@@ -69,6 +71,7 @@ interface OwnProps {
   onUpdateCodeStatus: (status: number) => void;
   onDownloadCode: () => void;
   onUploadCode: () => void;
+  onUpdateBitmap: (bitmap: number) => void;
```

Comment:
We can rename `onUpdateBitmap` to `onUpdateKeyboardBitmap` and `bitmap` to `keyboardBitmap`